### PR TITLE
Feat new UI receive

### PR DESCRIPTION
**Pull request by yanguoyu**
1. Receive with HD wallet

https://user-images.githubusercontent.com/12881040/228713251-8d35ab92-7148-46a8-a4ab-5c9ab4314032.mov

2. Receive with Hardware wallet

https://user-images.githubusercontent.com/12881040/228713203-45ec1b49-de2f-4648-b43a-444784dc2eb2.mov

3. Receive with the dark theme

https://user-images.githubusercontent.com/12881040/229423772-73c90510-347d-4373-81aa-8c690f15a1ba.mov

### CODE REVIEW

**Comment by devchenyan** [comment]
Maybe upload a video to display the dark theme @yanguoyu

File: packages/neuron-ui/src/components/VerifyHardwareAddress/index.tsx [hunk 1]
```diff
@@ -149,113 +145,85 @@ const VerifyHardwareAddress = ({ address, wallet, onDismiss }: VerifyHardwareAdd
...
+  }, [deviceInfo, address, userInputStatus, ensureDeviceAvailable, t])
 
   useDidMount(() => {
     // @ts-ignore
     dialogRef.current?.showModal()
     ensureDeviceAvailable(deviceInfo)
   })
 
-  const dialogClass = `${styles.dialog}`
+  // const onCopyAddress = useCallback(() => {
```
Comment (JeffreyMa597) [inline packages/neuron-ui/src/components/VerifyHardwareAddress/index.tsx:178]: Should `TODO`  or remove?

File: packages/neuron-ui/src/widgets/Table/table.module.scss [hunk 2]
```diff
@@ -53,10 +53,59 @@
...
+      }
+      &::-webkit-scrollbar-thumb {
+        width: 6px;
+        background: #CCCCCC;
+        border-radius: 48px;
+      }
+      @media (prefers-color-scheme: dark) {
+        &::-webkit-scrollbar-thumb {
+          background: #000;
+          border: 1px solid #343E3C;
```
Comment (JeffreyMa597) [inline packages/neuron-ui/src/widgets/Table/table.module.scss:83]: How about making the color global?

File: packages/neuron-ui/src/widgets/Table/table.module.scss [hunk 3]
```diff
@@ -53,10 +53,59 @@
   }
 }
 
+.fixedTableRoot {
+  &[data-have-head="true"] {
+    .table {
+      height: calc(100% - 52px);
```
Comment (JeffreyMa597) [inline packages/neuron-ui/src/widgets/Table/table.module.scss:59]: should we make the `52px` to be a param?

**Review by devchenyan — APPROVED** [review]

**Review by JeffreyMa597 — APPROVED** [review]

**Pull request merged**